### PR TITLE
fix: dropped sign in module for home page and revised page content

### PR DESCRIPTION
**Pull request by davidruvolo51**
This merge request fixes a few aspects with the ERN application.

- [x] dropped sign in module for the home page and app-footer component
- [x] revised "quick links" section: dropped sign in link and added photo for the dashboard card

### CODE REVIEW

File: apps/ern-reconnet/src/components/AppFooter.vue [hunk 1]
```diff
@@ -21,12 +21,7 @@
         <div class="footer-column footer-links">
           <p><strong>For Members</strong></p>
           <UnorderedList listType="none">
-            <li>
-              <button class="toggle" @click="updateShowSigninForm">
-                <span>Sign in</span>
-              </button>
-              <MolgenisSignin v-if="showSigninForm" @cancel="updateShowSigninForm"/>
-            </li>
+            <li> </li>
```
Comment (connoratrug) [inline apps/ern-reconnet/src/components/AppFooter.vue:24]: why the empty list

File: apps/ern-reconnet/src/views/home-page.vue [hunk 2]
```diff
@@ -39,15 +39,9 @@
         <LinkCard id="quicklink-documents" :imageSrc="DocumentsImage">
           <router-link :to="{ name: 'documents' }">Documents</router-link>
         </LinkCard>
-        <LinkCard id="quicklink-dashboard">
+        <LinkCard id="quicklink-dashboard" :imageSrc="DashboardImage">
```
Comment (connoratrug) [inline apps/ern-reconnet/src/views/home-page.vue:42]: use direct asset loader ?

**Review by connoratrug — APPROVED** [review]
some small code notes, see comments

File: apps/ern-reconnet/src/components/AppFooter.vue (hunk 1 above)
Comment (davidruvolo51) [reply apps/ern-reconnet/src/components/AppFooter.vue]: Removed from the component.

File: apps/ern-reconnet/src/views/home-page.vue (hunk 2 above)
Comment (davidruvolo51) [reply apps/ern-reconnet/src/views/home-page.vue:42]: I will fix this in the next release.

**Pull request merged**